include/types.h

```diff
@@ -24,13 +24,37 @@
 #define SCREEN_CENTER_X (SCREEN_WIDTH  / 2)
 #define SCREEN_CENTER_Y (SCREEN_HEIGHT / 2)
 
+typedef union {
+    struct PACKED {
+        /*0x00*/ u8 hi; // HI byte.
+        /*0x01*/ u8 lo; // LO byte.
+    }; /*0x02*/
+    u16 raw;
+} HiLo16; /*0x02*/
+
+typedef union {
+    struct PACKED {
+        /*0x00*/ u16 hi; // HI byte.
+        /*0x01*/ u16 lo; // LO byte.
+    }; /*0x02*/
+    u32 raw;
+} HiLo32; /*0x02*/
+
+typedef union {
+    struct PACKED {
+        /*0x00*/ u32 hi; // HI byte.
+        /*0x01*/ u32 lo; // LO byte.
+    }; /*0x02*/
+    u64 raw;
+} HiLo64; /*0x02*/
+
 struct Config {
-    f32 audioFrequency;
+    /*0x00*/ f32 audioFrequency;
```

Comment:
these are just formatting changes? the 0x04 stuff is not really relevant for hacking, they are just a remnant of decomp. idk why youd go out of your way to add more of them in

Reply:
Reverted